stable/pgadmin/values.yaml

```diff
@@ -71,8 +71,9 @@ ingress:
 ## Values in here get injected as environment variables
 ##
 env:
-  email: chart@example.local
-  password: SuperSecret
+  PGADMIN_DEFAULT_EMAIL: chart@example.local
+  PGADMIN_DEFAULT_PASSWORD: SuperSecret
+  PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION: 'False'
```

Comment:
Please use lower case, and just `# enhanced_cookie_protection: False`

e.g.:
```
## pgAdmin startup configuration
## Values in here get injected as environment variables
##
env:
  email: chart@example.local
  password: SuperSecret
  enhanced_cookie_protection: False
```

Reply:
Would it perhaps be nice to add a comment above enhanced_cookie_protection with a link towards its documentation?

Reply:
done